.github/workflows/generate.yml

```diff
@@ -53,3 +53,24 @@ jobs:
           files: "fleet/*.sqlite.gz"
           tag_name: ${{ env.ETAG }}
           token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Slack Notification
+        # if: failure() # will uncomment once tested.
+        uses: slackapi/slack-github-action@v1.18.0
+        with:
+          payload: |
+            {
+              "text": "${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head.html_url }}",
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "NVD db generation result: ${{ job.status }}\nhttps://github.com/fleetdm/nvd/actions/runs/${{  github.run_id }}\n${{ github.event.pull_request.html_url || github.event.head.html_url }}"
+                  }
+                }
+              ]
+            }
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_CORE_ENGINEERING_WEBHOOK_URL }}
```

Comment:
@zwass Please clone the secret from the fleet repo into this repository. Let me know when it's done so I can manually trigger and test.

Reply:
It looks like this secret should be available. Please let me know if you encounter issues.